script/Deploy.sol

```diff
@@ -57,6 +59,23 @@ contract Deploy is Script {
         vm.stopBroadcast();
     }
 
+    function deployOffchainAssetReceiptVaultFactory(uint256 deploymentKey) internal {
```

Comment:
i don't really know what we're trying to achieve here when we already have `deployImplementations`?